services/patient-manager/app/api/db.py

```diff
@@ -34,12 +35,13 @@ class Patient(Base):
 
     patient_id: Mapped[int] = mapped_column(primary_key=True)
 
-    sex: Mapped[str] = mapped_column(nullable=False)
-    name: Mapped[str] = mapped_column(nullable=False)
-    birth_date: Mapped[str] = mapped_column(nullable=False)
+    first_name: Mapped[str] = mapped_column(nullable=False)
+    last_name: Mapped[str] = mapped_column(nullable=False)
+    birth_date: Mapped[datetime.date] = mapped_column(nullable=False)
+    sex: Mapped[Literal["MALE", "FEMALE", "OTHER", "NOT_GIVEN"]] = mapped_column(nullable=False)
```

Comment:
See comment regarding exam status. I would suggest to use a pydantic model here (example can be found in task type)

Reply:
Done